.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Prime Data Hub
+
+on:
+  push:
+    branches: [ master, production ]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Environment Variables
```

Comment:
This chunk will grab all of the secrets (depending on environment) and set them as environment variables during the release.